styles/main.css

```diff
@@ -0,0 +1,129 @@
+* {
+	box-sizing: border-box;
+	padding: 0;
+	margin: 0;
+}
+
+html {
+	scroll-behavior: smooth;
+}
+
+.grid-container {
+	display: grid;
+	gap: 0;
+	grid-template-areas:
+		'h m m m'
+		'n m m m'
+		'n m m m'
+		'n f f f';
+}
+
+header {
+	grid-area: h;
+	height: 10vh;
+	width: 20vw;
+	padding-top: 4%;
+	display: flex;
+	justify-content: center;
+	position: sticky;
+	top: 0;
+}
+
+nav {
+	grid-area: n;
+	width: 20vw;
+	display: flex;
+	justify-content: center;
+	font-family: 'Roboto', sans-serif;
+	white-space: nowrap;
+}
+
+.sticky {
+	position: sticky;
+	top: 0;
+	height: 100vh;
+	background-color: rgba(205, 205, 205, 0.665);
+}
+.navList {
+	list-style: none;
+	padding-top: 10%;
+}
+
+.navItem {
+	margin-bottom: 5%;
```

Comment:
Same as my comment on the top-padding property on Line 25, I would recommend using `rem` or `em` based on your preference.